src/config/rollup.config.js

```diff
@@ -9,10 +9,9 @@ const {
 } = require('@rollup/plugin-node-resolve')
 const replace = require('@rollup/plugin-replace')
 const camelcase = require('lodash.camelcase')
-const {terser} = require('rollup-plugin-terser')
-const nodeBuiltIns = require('rollup-plugin-node-builtins')
-const nodeGlobals = require('rollup-plugin-node-globals')
 const omit = require('lodash.omit')
+const nodePolyfills = require('rollup-plugin-polyfill-node')
```

Comment:
I also replaced `rollup-plugin-node-builtins`& `rollup-plugin-node-globals` with `rollup-plugin-polyfill-node` as that's what the @rollup team [recommends](https://rollupjs.org/guide/en/#warning-treating-module-as-external-dependency) & the @babel team did as well in https://github.com/babel/babel/pull/11098.